bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -52,8 +53,9 @@ public boolean allowedInCommit() {
     }
 
     @Override
-    public boolean allowedInPullRequest() {
-        return false;
+    public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath,
+                CommandInvocation command, List<Comment> allComments, PrintWriter reply) {
+        reply.println("The command `backport` can not be used in a pull request that has not yet been integrated.");
```

Comment:
Shouldn't the message here be the one suggested in the Bug?

"(!) You cannot issue `/backport` on a pull request. Instead, give that command on the actual commit you want to backport."

Reply:
I want the`/backport` command can be used in the PRs which hava been integrated. So the message `You cannot issue /backport on a pull request` seems not accurate. But the current message seems not accurate too. Looking for a better description and your idea.

Reply:
Additional information: When the PR has been integrated and the `backport` command is used in this integrated PR, the another `handle` method  instead of this method will be invoked so that the `backport` command can be processed.

Reply:
In this patch, it can be used at PRs which had been integrated. This is a mian feature of this patch.

Reply:
Oh, I see now that PullRequestCommandWorkItem will call the handle method for commit if the PR has been integrated. That was rather surprising behavior. In that case, please align the reply with the one in PullRequestCommandWorkItem:

printer.print("The command `");
printer.print(command.name());
printer.println("` can only be used in a pull request that has been integrated.");

Reply:
> please align the reply with the one in PullRequestCommandWorkItem

Fixed.

> Also, if that's the case, please change the bug and PR title to make this clear. That's the main feature being added here.

I revise the title just now.